iwyu_include_picker.cc

```diff
@@ -95,21 +95,29 @@ const IncludeMapEntry libc_symbol_map[] = {
   // an option for this type.  That's the preferred #include all else
   // equal.  The visibility on the symbol-name is ignored; by convention
   // we always set it to kPrivate.
+  { "_POSIX_VDISABLE", kPrivate, "<unistd.h>", kPublic },
```

Comment:
Looks like we have this whole map duplicated in https://github.com/include-what-you-use/include-what-you-use/blob/7b3205c3ca548463edf502045fea9c6c8d9d43c3/gcc.libc.imp -- can you make the same changes there for now?

At some point it would obviously be better to generate this whole thing from the same source, but I'd prefer to take some cheap wins first, and then think of the whole POSIX generator separately.

(the external mapping is useful when running with `-Xiwyu --no_default_mappings` e.g. for some baremetal setup, and then being able to selectively add back the builtin sets with e.g. `-Xiwyu --mapping_file=gcc.libc.imp`)

(I'm also aware that the file is probably misnamed, I think the libc is not directly associated with GCC)

Reply:
Did you mean to link to `gcc.symbols.imp`? The other file has header mappings not symbol ones.

Reply:
Oof, yes -- sorry! I meant `gcc.symbols.imp`.